### ISSUE TITLE
Decreasing the number of columns in the "predation-accessibility.csv" file 

### ISSUE THREAD

**Issue by agruss2** [jhpoelen/fb-osmose-bridge#126]
@jhpoelen 
The parameters that are provided in the "predation-accessibility.csv" file are the accessibility coefficients of focal and background organisms to focal organisms. 
This information is provided in the form of a matrix, where a given line provides coefficients reflecting the accessibility of a given focal organism or of a given background organism to the different bakcground organisms represented in the OSMOSE model.
Therefore, in the "predation-accessibility.csv" file, there should be more lines than columns. 
Here is an example:
I just ran a test with the bridge between FishBase/SeaLifeBase and OSMOSE, where I queried parameter values for an OSMOSE model for the Iceland Shelf/Sea ecosystem. 
The phytoplankton and zooplankton groups were defined as background functional groups (as is the case for all OSMOSE models). 
I also defined the "invertpelagicsm" functional group as a background functional group. 
Then, I got the following "predation-accessibility.csv" file (provided here in the .xlsx format): 
[predation-accessibility.xlsx](https://github.com/jhpoelen/fb-osmose-bridge/files/1118180/predation-accessibility.xlsx)
For this "predation-accessibility.csv" file to be useable in OSMOSE, the last three columns that are currently provided (for phytoplankton, zooplankton, and invertpelagicsm) should not appear in the file. 

--> Could you please correct for this? 
Once you are done, please let me know so that I can run a new test with the bridge between FishBase/SeaLifeBase and OSMOSE and, eventually, close the present issue. 
Many thanks.

**Comment by jhpoelen** [comment]
Thanks for your detailed description. I've implemented the changes as suggested. Please confirm.

**Comment by agruss2** [comment]
@jhpoelen I just ran a test and can confirm that the present issue has been solved; I can therefore close it now. Many thanks.
